manage

```diff
@@ -620,6 +657,16 @@ startServices() {
     fi
   fi
 
+  if [[ -z ${REDIS_CLUSTER_URL_CONFIG} ]]; then
```

Comment:
Redis should probably not be started from here. Either it should be `./manage` command line triggered or it should be started independently and before running `./manage`.  Other services are run that way -- notably `orb`.  I ran a test using the `acapy` backchannel, and redis was started.

Reply:
I'll look into this, I thought I had replicated the other patterns, but clearly not, thanks.

Reply:
I agree, it will be unused in most circumstances, it is not started by default anymore. Will need to consider how to notify the user of the situation where the that they are using an image that needs redis but isn't running\